canvas_account_admin_tools/views.py

```diff
@@ -55,40 +55,27 @@ def lti_launch(request):
 
 
 @login_required
-@lti_role_required(const.ADMINISTRATOR)
-@lti_permission_required(settings.PERMISSION_ACCOUNT_ADMIN_TOOLS)
+@lti_permission_required(PERMISSIONS['canvas_account_admin_tools'])
```

Comment:
Hmm, thinking aloud. We would need the dict format to enable this right? If we change the TOOL_PERMISSIONS to a list, we loose the ability to selectively pick the appropriate permissions for the decorator here and for the different tools below?